ui/src/components/CardListItem.vue

```diff
@@ -1,21 +1,168 @@
 <template>
-    <div class="card">
-        <img class="card-img-top" src="" alt="Card image cap" />
-        <div class="card-body">
-            <h5 class="card-title">Card Title</h5>
-            <p class="card-text">Sample result</p>
-            <a href="#" class="btn btn-primary">Go Somewhere</a>
+    <div v-if="locations" class="ui fluid card">
+        <div class="content">
+            <h5 class="header">Locations</h5>
+            <p class="card-text">Source:</p>
+            <canvas id="location-chart"></canvas>
+        </div>
+    </div>
+    <div v-else-if="employmentTypes" class="ui fluid card">
+        <div class="content">
+            <h5 class="header">Employment Types</h5>
+            <p class="card-text">Source:</p>
+            <canvas id="employmentType-chart"></canvas>
+        </div>
+    </div>
+    <div v-else-if="skills" class="ui fluid card">
+        <div class="content">
+            <h5 class="header">Skills</h5>
+            <p class="card-text">Source:</p>
+            <canvas id="skill-chart"></canvas>
+        </div>
+    </div>
+    <div v-else-if="isRemote" class="ui fluid card">
+        <div class="content">
+            <h5 class="header">Remote Eligible</h5>
+            <p class="card-text">Source:</p>
+            <canvas id="remote-chart"></canvas>
         </div>
     </div>
```

Comment:
Again, shouldn't rely on specific data entry like this. There's too much duplication and not extendable with variable structure of data.
Try to organize the data into an array and then use `v-for`.